### PR TITLE
All features

### PR DESCRIPTION
**Pull request by Kakalanp**
- Adding Linters.

- Adding Git ignore.

- Adding Coco goose fonts (.otf).

- Adding Coco goose fonts (.ttf).

- Adding fonts.css.

- Adding styles and styles desktop CSS files.

- Adding Home and About HTML pages.

- Adding background and foreground images.

- Adding MIT.md.

- Adding README.md.

Loom video presenting the code: https://www.loom.com/share/9194eff233524101ba1fc5db54e7039b

### CODE REVIEW

File: About.html [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+            <ul class="menu">
+                <li><a class="active" href="About.html">About</a></li>
+                <li><a>Program</a></li>
+                <li><a>Join</a></li>
+                <li><a>Sponsors</a></li>
+                <li><a>News</a></li>
+                <li><a class="cc-campaign">CC Campaign</a></li>
+            </ul>
+            <a class="hamburger mobile" href="#">≡</a>
+        </nav>
```
Comment (Hamzaoutdoors) [inline About.html:43]: - [x] Good job implementing the navbar feature, it's working perfect, but there is a small issue : 
 - When I am on the `About` page in the mobile version I can't redirect to the home page using the navbar, Kindly try to work on this issue, as you can in every website those little details are so important 
 
about

File: index.html [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+        </nav>
+    </section>
+
+    <main>
+        <div class="main">
+            <h2>Hello future developer!</h2>
+            <h1 class="main-title">HTML Course is here!</h1>
+            <div class="spit-info"><p>Learn to program in HTML in only a month! With experts from around the world to guide you though the course!</p></div>
+            <div class="starting-date"><p>2022.03.01(Mon) ~ 2022.03.31(Thu)</p></div>
+        </div>
```
Comment (Hamzaoutdoors) [inline index.html:49]: - [x] Please try to stick the design as much as possible, you can give your project some padding to look more clear for the user

**Current mobile menu**           |  **Original design**
:-------------------------:|:-------------------------:
  |

**Review by Hamzaoutdoors — CHANGES_REQUESTED** [review]
Hey @Kakalanp, I am **Hamza** your reviewer for this PR 😸 

_You did a great job, I really enjoyed surfing on your fancy website 😃_ 

## To Highlight
✔️ The repo looks professional
✔️ Git/GitHub flow best practices have been followed
✔️ There are no linter errors
✔️ You have a descriptive README
✔️ Wonderful UI design and good choice of theme🥇 
✔️ Great Job building your project with **Mobile-First**

There are some issues that you still need to work on to go to the next project but you are almost there!

image

## Required Changes ♻️

_See the inline comments below for more required changes 😃_  

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [OTIOANAL ] 

- I recommend you too to regroup your `JavaScript` and `CSS` files in folders i.e. `script` and `styles` folder, what do you think, with that your root project will be more organized 😸 
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (Kakalanp) [reply index.html:49]: I just added some padding, as you suggested https://github.com/Kakalanp/Capstonproj-Conference-page/blob/0fe222a7dd22cd537af97f720b6aa0831dcd83e3/CSS/Styles.css#L134 What do you think of it now?

File: About.html (hunk 1 above)
Comment (Kakalanp) [reply About.html:43]: The last version now does this, moving the logo inside the mobile menu on-open: https://github.com/Kakalanp/Capstonproj-Conference-page/blob/0fe222a7dd22cd537af97f720b6aa0831dcd83e3/Js/script.js#L71-L73

And removing it on-close: https://github.com/Kakalanp/Capstonproj-Conference-page/blob/0fe222a7dd22cd537af97f720b6aa0831dcd83e3/Js/script.js#L51-L54

This is how it looks in the page now: 
image

**Review by ReshyResh — CHANGES_REQUESTED** [review]
# Hi @Kakalanp , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️

- [ ] You've done well implementing the logo button to return to the homepage, but it seems like on the mobile version it isn't properly centered, please make sure to address this issue.

|   |
| ------------- |
|   |

- [x] You're currently missing the footer on both the index and about page. The sponsors section is not the footer, please make sure to stick to the design and add the required footer.
- [x] The featured speakers section has a different layour from the template, the elements are currently laid horizontally on the page, but they should be positioned on rows of two elements each. The mobile version is okay.

| Your section  | Template section |
| ------------- | ------------- |
|   |   |


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Kakalanp** [comment]
> * You've done well implementing the logo button to return to the homepage, but it seems like on the mobile version it isn't properly centered, please make sure to address this issue.
> 
> 

In my screen it doesn't seem off, I can't quite put my finger in what could be causing that glitch.

**Comment by Kakalanp** [comment]
> > * You've done well implementing the logo button to return to the homepage, but it seems like on the mobile version it isn't properly centered, please make sure to address this issue.
> > 
> > 
> 
> In my screen it doesn't seem off, I can't quite put my finger in what could be causing that glitch.

I did some research, testing, and finally replicated the issue. The only browser that didn't show the icon aligned was **Mi Browser** from Xiaomi, a friend of mine found the bug by accident, but I can't find any documentation / solution to the problem. If anyone has a suggestion, please share your thoughts on the topic. Thanks!

File: About.html [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+    <div class="succesful-students">
+        <h2>See succesful students stories</h2>
+        <p>Take a look at the last two students who graduated from HTML Course.</p>
+        <div class="student"><a href="">Name</a><p>Student from Argentina!</p></div>
+        <div class="student"><a href="">Name</a><p>Student from Ghana!</p></div>
+    </div>
+    </main>
+    
+    <footer class="about-footer">
```
Comment (Deepakdanger) [inline About.html:71]: - [x] **Required** In the mobile version the footer should match the original website, the logo should be present on the left side & the background should be white on the mobile screen.
- Your design 
Home footer
rwer
About page footer 
xasx

- Original website 
klklk

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @Kakalanp, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by oloomoses — APPROVED** [review]
Hi @Kakalanp, 

Well done on making all the required changes.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

_Slack: Oloo Moses_
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**